### PR TITLE
reorg of docs (still needs content)

### PR DESCRIPTION
**Pull request by atomicsamurai**
#185

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,24 +4,39 @@ This is the ForgeROck DO - or short frodo - command line interface, a CLI to man
 
 Frodo is the successor to field tools like [amtree.sh](https://github.com/vscheuber/AM-treetool), [fidc-debug-tools](https://github.com/vscheuber/fidc-debug-tools), and ForgeRock-internal utilities.
 
+## Quick start
+
+1. Download you platform specific binary zip from the [release page](https://github.com/rockcarver/frodo/releases) and unzip it to a directory.
```
Comment (vscheuber) [inline README.md:9]: Download _your_ platform specific binary _archive_
or
Download _the_ platform specific binary _archive_

File: docs/commands/idm.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# `idm` command
```
Comment (vscheuber) [inline docs/commands/idm.md:1]: My main concern with all these pages is that they get out of date so quickly. We change commands, add new ones, and doc is always behind.

Also, most of what we have in these command pages seems to be basically the same as what a user could get from running the frodo help function for a command.

Maybe if instead we gave away some secrets or tips?

**Review by vscheuber — CHANGES_REQUESTED** [review]
I love the breaking down into individual files. I wonder if we should remove all the command docs, though. Frodo help should cover that. We should focus on install and dev process and if our help needs improvement, let's fix it. But I fear static command doc will be stale quickly.

File: docs/commands/idm.md (hunk 2 above)
Comment (atomicsamurai) [reply docs/commands/idm.md:1]: I had the same feeling when looking at the command specific content. That is why I didn't merge the request and wanted another opinion. I agree that the cli help should be only source of that information and if needed we can make it friendlier/verbose. But, I also think that we should keep the list of supported features updated in the README. Thoughts?

**Pull request merged**